### ISSUE TITLE
[Feature request] Support for media alt text on Mastodon accounts

### ISSUE THREAD

**Issue by contraexemplo** [TwidereProject/Twidere-Android#1007]
- [v.2.0.0](https://github.com/tootsuite/mastodon/releases/tag/v2.0.0) introduced the ability to [add alt text to media](https://github.com/tootsuite/mastodon/pull/5123) on Mastodon. Even though there is already a way to do this on Twitter accounts with Twidere, it doesn't seem to be compatible with Mastodon.
- This is an important feature regarding accessibility. v2.0.0 was published 9 days ago and the last update for Twidere was released on October 11, my guess it isn't implemented yet since I posted [this toot](https://cybre.space/@Anna/98900682466133609) from Twidere for Android and the alt text is lacking.

**Comment by contraexemplo** [comment]
Hey @mariotaku, thank you for implementing this.

**Comment by contraexemplo** [comment]
@mariotaku Some friends of mine and I are having problems with the alt text feature. Should I open a new issue or reopen this one?

**Comment by mariotaku** [comment]
@contraexemplo You can open a new issue, thanks.
